pkgs/modules/c/default.nix

```diff
@@ -10,7 +10,7 @@ let
   dap-cpp = pkgs.callPackage ../../dap-cpp { };
   dap-cpp-messages = import ../../dap-cpp/messages.nix;
 
-  clang-version = lib.versions.majorMinor clang.version;
+  clang-version = lib.versions.major clang.version;
```

Comment:
Thought: should we have different modules for the different C standards? I'd ship C99 in its own module vs C11 vs...

Reply:
I think that's a flag to clang. Maybe should be a configuration? But we don't have a way for users to configure things currently...

Reply:
gotcha, we should look into adding something for this...